RedfishPkg/RedfishCredentialDxe/RedfishCredentialDxe.c

```diff
@@ -13,6 +13,8 @@
 
 #define REDFISH_VERSION_DEFAULT_STRING  L"v1"
 
+extern int  _fltused;

```

Comment:
I do not think this is required.  No code actually references this global.  It is intrinsic from compiler